tests/hub-local/test_update_schema_in_hub.py

```diff
@@ -63,6 +68,8 @@ def test_update_schema_in_hub(setup_instance):
         "model_name": "artifact",
         "schema_name": "core",
         "is_link_table": False,
+        "is_primary_key": False,
+        "is_editable": False,
```

Comment:
@falexwolf It became not editable after I updated the PR, does it make sense?

Reply:
Yes, many fields are now no longer editable.